### PR TITLE
New gas customization: basic tab content w/ gas price button group

### PR DESCRIPTION
**Pull request by danjm**
No connections to dynamic data yet, will be done in another PR.

basictabstatic

### CODE REVIEW

File: ui/app/components/gas-customization/gas-modal-page-container/index.scss [hunk 1]
```diff
@@ -51,5 +52,7 @@
   &__info-row--fade {
     background: white;
     color: $dusty-gray;
+    border-top: 1px solid $mischka;
+    margin-top: 22px;
   }
 }
```
Comment (alextsg) [inline ui/app/components/gas-customization/gas-modal-page-container/index.scss]: nit: newline

File: ui/app/css/itcss/settings/variables.scss [hunk 2]
```diff
@@ -58,6 +58,8 @@ $linen: #fdf4f4;
 $oslo-gray: #8C8E94;
 $polar: #fafcfe;
 $blizzard-blue: #bfdef3;
+$spindle: #c6dded;
```
Comment (alextsg) [inline ui/app/css/itcss/settings/variables.scss]: We have a `$spindle` already at line 46. Is that `$spindle` too different from the color used in the mock?

File: ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/basic-tab-content.component.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  static propTypes = {
+    gasPriceButtonGroupProps: PropTypes.object,
+  }
+
+  render () {
+    return (
+      <div className="basic-tab-content">
+        <div className="basic-tab-content__title">Suggest gas fee increases</div>
```
[the remark this answers is not in the document]
Comment (danjm) [reply ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/basic-tab-content.component.js]: I'm going to wait until design and copy is finalized before making all of these translations. It is possible some of this copy will be deleted before this gets to master. I made an issue for this https://github.com/metamask/metamask-ui/issues/27

**Comment by danjm** [comment]
@alextsg All issues fixed (minus translations, see previous comment)

**Review by alextsg — APPROVED** [review]

**Pull request merged**